client/components/Forms/EventForm.js

```diff
@@ -163,13 +187,17 @@ class EventForm extends Component {
   )
 
   render() {
-    const { errors } = this.state;
+    const {
+      errors,
+      categoryError,
+    } = this.state;
     const {
       formId,
       formData,
+      categories,
     } = this.props;
     const { title, description, venue } = this.state.formData
-
+    const catClass = categoryError ? 'category-label category-error' : 'category-label';
```

Comment:
A more descriptive name is `categoryClass`

Reply:
💯